tests/test_room_api.py

```diff
@@ -115,6 +119,23 @@ def test_invalid_video_codec(self, room_api):
         with pytest.raises(ValueError):
             room_api.create_room(max_peers=MAX_PEERS, video_codec="h420")
 
+    def test_custom_room_id(self, room_api):
+        room_id = str(uuid.uuid4())
+        _, room = room_api.create_room(room_id=room_id)
+
+        assert room == Room(
+            components=[],
+            config=RoomConfig(
+                room_id=room.id,
+                max_peers=None,
+                video_codec=None,
+                webhook_url=None,
+            ),
+            id=room_id,
+            peers=[],
+        )
+        assert room in room_api.get_all_rooms()
```

Comment:
What do you think about a test where we would try to create a room with already existing `room_id`?